libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/guidance/signboard/api/MapboxSignboardApi.kt

```diff
@@ -21,7 +22,8 @@ import kotlinx.coroutines.launch
  * @property accessToken String
  */
 class MapboxSignboardApi(
-    private val accessToken: String
+    private val accessToken: String,
+    private val options: MapboxSignboardOptions
```

Comment:
Should this be defaulted to `MapboxSignboardOptions.Builder().build()` (`@JvmOverloads for Java users) to avoid forcing users to give in `MapboxSignboardOptions`?

Reply:
Also should the `accessToken` be a part of `MapboxSignboardOptions`? 🤔

Reply:
I would add the default, but I will leave the access token as is to maintain consistency with other UI components API